devsetup/tripleo/undercloud.conf.j2

```diff
@@ -30,13 +30,13 @@ local_mtu = {{ interface_mtu }}
 # Undercloud services. Only used with SSL. (string value)
 # Deprecated group/name - [DEFAULT]/undercloud_public_vip
 #undercloud_public_host = 192.168.24.2
-undercloud_public_host = 192.168.122.122
+undercloud_public_host = 192.168.122.99
```

Comment:
what is the reason for this change? was this ip unused until now, so we had a placeholder value?

Reply:
ack, thanks I assumed that was the case but wanted to clarify